test/chainsaw/testmetrics/dynatrace-provider/chainsaw-test.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: dynatrace-provider-auth
+spec:
+  namespaceTemplate:
+    metadata:
+      annotations:
+        keptn.sh/lifecycle-toolkit: enabled
+  steps:
+    - name: step-00
+      try:
+        - apply:
+            file: mock-server.yaml 
+        - apply:
+            file: 00-install.yaml   
+        - assert:
+            file: 00-assert.yaml
```

Comment:
```suggestion
            file: 00-assert.yaml
            
```

Reply:
Same here, I don't get the suggested change